.circleci/config.yml

```diff
@@ -145,6 +145,10 @@ commands:
       - run:
           name: Verify the build environment
           command: ./libs/verify-desktop-environment.sh
+      # Install SQLite development libraries
+      - run:
+          name: Install SQLite development libraries
+          command: sudo apt-get install libsqlite3-dev
```

Comment:
Also seemed weird to me. This commit was just to test if the tests would run through (and they did). I'll do some more digging to see why. It's just that `nimbus-sdk` tests fail because of missing `sqlite-dev` in the CI.